ui/packages/catalog-ui-search/src/main/webapp/react-component/sharing/presentation.tsx

```diff
@@ -126,7 +126,7 @@ const render = (props: Props) => {
                   {item.category === Category.User ? (
                     <EditableLabelText
                       value={item.value}
-                      placeholder="user@example.com"
+                      placeholder="user id"
```

Comment:
This is a bit unfortunate as "user id" doesn't really mean anything to the actual user

Reply:
✏️ `User Id`? Like in the master PR.